tutorial/transport/linopy_model.py

```diff
@@ -0,0 +1,126 @@
+# m = linopy.Model()
```

Comment:
Suggest to rename this file to "dantzig_model_linopy.py" (because we may have other models later).

Reply:
Done, but please note that we now have
```python
from tutorial.transport.dantzig_model_linopy import (
    create_dantzig_model,
    read_dantzig_solution,
)
```
where I read `transport` as equivalent to `dantzig` and thus see a lot of repetition.

Reply:
Don't think that this is too bad...